Increase RPMBUFSIZ to increase the possibility to find magic compression identifier
http://bugs.gentoo.org/187219#c13

--- rpmoffset.c
+++ rpmoffset.c
@@ -10,7 +10,7 @@
 /* chunk of RAM right away so that we have enough.  Yeah, horrible */
 /* quick and dirty implementation, but hey -- it gets the job done. */
 
-#define RPMBUFSIZ 2097152
+#define RPMBUFSIZ 3145728
 
 main()
 {
